static/app/stores/pageFiltersStore.tsx

```diff
@@ -117,45 +101,46 @@ interface PageFiltersStoreDefinition
 }
 
 const storeConfig: PageFiltersStoreDefinition = {
-  selection: getDefaultSelection(),
-  pinnedFilters: new Set(),
-  desyncedFilters: new Set(),
-  shouldPersist: true,
-  hasInitialState: false,
+  state: {
```

Comment:
could be using the `private` or `protected` keyword so callers know to continue to use `getState()`

```suggestion
  private state: {
```

Reply:
internally it seems like there's a mix of calling `this.state` and `this.getState()`. but nbd.

Reply:
ideally, but i think those only exist in classes

Reply:
it is readonly now at least https://github.com/getsentry/sentry/blob/master/static/app/stores/types.tsx#L25